sql/ts/src/skdb-cli.ts

```diff
@@ -490,7 +490,9 @@ const localRepl = async function () {
       console.log(".schema -- Output the schema.");
       console.log(".table-schema <table> -- Output the schema <table>.");
       console.log(".view-schema <view> -- Output the schema for <view>.");
-      console.log(".mirror <table1> <table2> ... -- Mirror the remote tables.");
+      console.log(
+        ".mirror <table1> <schema1> <table2> <schema2> ... -- Mirror the remote tables.",
+      );
```

Comment:
This is going to be a pain in the ass when developing, testing, debugging. Can we do something where you can say "just give me the latest schema, it's cool, this isn't for application code". Maybe if you send a special string like "(*)"? I'm open to whatever but I think it'll be worth it for user driven tooling that doesn't have app logic written against a static schema.